Orange/widgets/data/owcorrelations.py

```diff
@@ -384,6 +392,7 @@ def commit(self):
             self.Outputs.data.send(self.data)
             self.Outputs.features.send(None)
             self.Outputs.correlations.send(None)
+            self.info.set_output_summary(self.info.NoOutput)
```

Comment:
This it not correct info when `self.cont_data is None` and `self.data is not None`.